modules/lagoon/govcms_lagoon/govcms_lagoon.info

```diff
@@ -8,3 +8,4 @@ dependencies[] = fast_404
 dependencies[] = clamav
 dependencies[] = robotstxt
 dependencies[] = lagoon_logs
+dependencies[] = govcms_security
```

Comment:
GovCMS D7 is using govcms_account_security, and GovCMS D8&9 is using govcms_security. So maybe we have to update the module name here. Otherwise, it is a good for security :)

Reply:
Perhaps GovCMS 7 needs a "stub" govcms_security module, which simply has govcms_account_security as a dependency? 🤷